SRC/Peripheral/inc/CH56x_flash.h

```diff
@@ -0,0 +1,27 @@
+/********************************** (C) COPYRIGHT *******************************
+* File Name          : CH56x_flash.h
+* Author             : Hans Baier
+* Version            : V1.1
+* Date               : 2022/08/02
+* Description        : Contains functions for reading and writing flash
+* Copyright (c) 2022 Hans Baier
+* SPDX-License-Identifier: Apache-2.0
+*******************************************************************************/
+#ifndef __CH56x_FLASH_H__
+#define __CH56x_FLASH_H__
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+#include "CH56xSFR.h"
+
+UINT8  FLASH_ROMA_ReadByte ( UINT32 addr );
```

Comment:
Could you replace the ugly UNIT8/UINT32 and so one by more standard (from stdint.h) uint8_t/uint32_t ?